### DISCUSSION TITLE
Is Mocking HEAD method APIs possible through specmatic

### DISCUSSION THREAD

**Discussion by DarshanS4444** [specmatic/specmatic#1011] [General]
Problem statement : Unable to stub responses for a HEAD method API. 
Description : We are trying to mock stub responses for download invoice PDF feature. The API used to return the downloaded invoice has method type HEAD, and content type as application/pdf in response header. The response body is in the form of inputstream. 

Getting the following error : 
An operation is not implemented: Method "HEAD" in scenario New scenario

Information required : Is it possible to stub API with method HEAD. Please let us know. Thank you.

**Comment by burdeamit** [comment]
Hey @DarshanS4444,
Thanks for reaching out. Sorry for the delayed response. 
We will discuss this issue internally and get back to you.
